Wikipedia/Code/TalkPageReplyComposeContentView.swift

```diff
@@ -283,22 +284,26 @@ class TalkPageReplyComposeContentView: SetupView {
         
         return attributedString
     }
+
+    private func evaluatePublishButtonEnabledState() {
+        publishButton.isEnabled = !replyTextView.text.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty
```

Comment:
Nice - working great! I didn't make the new topic screen this smart (trimming whitespaces)...do you mind improving it there as well?

Reply:
Sure thing - updated and ready for another look!